java/sage/Person.java

```diff
@@ -196,6 +235,14 @@ public boolean isPersonIdentical(Person testMe)
       for (int i = 0; i < awardNames.length; i++)
         if (awardNames[i] != testMe.awardNames[i])
           return false;
+      for (int i = 0; i < headshotUrls.length; i++)
+      {
+        if (headshotUrls[i].length != testMe.headshotUrls[i].length)
+          return false;
+        for (int j = 0; j < headshotUrls[i].length; j++)
+          if (headshotUrls[i][j] != testMe.headshotUrls[i][j])
```

Comment:
You're going to want to do an Array.equals comparison here. I got away with straight == in the above ones because I'm comparing Stringer objects which will then match because they are looked up from the DB table.